llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -828,7 +828,8 @@ struct AllocMatchInfo {
 };
 
 DenseMap<uint64_t, SmallVector<CallEdgeTy, 0>>
-memprof::extractCallsFromIR(Module &M, const TargetLibraryInfo &TLI) {
+memprof::extractCallsFromIR(Module &M, const TargetLibraryInfo &TLI,
+                            function_ref<bool(uint64_t)> IsPresentInProfile) {
```

Comment:
Do you think it's a good idea to add a default `IsPresentInProfile = [](uint64_t) { return true; }` so that we can skip the one-liners in the test?

Reply:
Yes.  Fixed.